pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -129,11 +131,11 @@ func (r ResourceLogsUnmarshaler) UnmarshalLogs(buf []byte) (plog.Logs, error) {
 	return l, nil
 }
 
-func getTimestamp(record azureLogRecord) (pcommon.Timestamp, error) {
+func getTimestamp(record azureLogRecord, format []string) (pcommon.Timestamp, error) {
```

Comment:
maybe a hint for you to reduce the code changes, use a var arg:
```suggestion
func getTimestamp(record azureLogRecord, formats ...string) (pcommon.Timestamp, error) {
```

Reply:
updated